transformers/announce.sh

```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env bash
+set -e
+
+function message() {
+    echo
+    echo -----------------------------------
+    echo "$@"
+    echo -----------------------------------
+    echo
+}
+
+if ! sentry-cli -V &> /dev/null
+then
+  curl -sL https://sentry.io/get-cli/ | bash
+fi
+
+ENVIRONMENT=$1
+if [ "$ENVIRONMENT" == "prod" ]; then
+TAG=latest
+elif [ "$ENVIRONMENT" == "staging" ]; then
+TAG=staging
+else
+   message UNKNOWN ENVIRONMENT
+   echo 'Allowed values for environment are "staging" or "prod"'
+   exit 1
+fi
+
+PROCESS=$2
+if [ -z "$PROCESS" ]; then
+    message NO PROCESS NAME
+    echo 'You must specify process name, such as vdb-execute (bash announce.sh <ENVIRONMENT> <PROCESS>).'
```

Comment:
nbd but maybe the example could be `vdb-oasis-execute` instead of `vdb-execute`

Reply:
Fixed - but more importantly this file is in the wrong place!